src/cli/commands/project.ts

```diff
@@ -16,6 +16,10 @@ export class Project extends Command {
       char: 'm',
       exclusive: ['json'],
     }),
+    projectDir: flags.string({
```

Comment:
Since we're on the `project` sub command let's just call this `dir`

In the description we should explain "the path to the root of your package, defaults to the current working directory" or something like that.